src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -56,11 +56,12 @@ export class SkyAvatarComponent {
     this._canChange = value;
   }
 
+  @Input()
+  public maxFileSize = 500000;
```

Comment:
now that it's an input, does there need to be any input checking? make sure it's not negative, make sure it's not too big, any tests that are associated with that? 

or is all that overkill?

Reply:
This is just a pass through to the existing input on the File drop component within. If any input checking is needed it should already be handled within that component 🙂 
https://github.com/blackbaud/skyux-avatar/blob/master/src/app/public/modules/avatar/avatar.component.html#L7

Reply:
it looks like that it eventually gets passed to the [`file-attachement.service`](https://github.com/blackbaud/skyux-forms/tree/1ca8d21fe11f781701031ce5c3edd579fa04a7b9/src/app/public/modules/file-attachment), which doesn't appear to have tests. maybe it's tested elsewhere. either way, doesn't seem like a concern of this pr. carry on.